docs/site/Best-practices.md

```diff
@@ -10,38 +10,25 @@ redirect_from:
 summary:
 ---
 
-{% include important.html content=" The API-first approach for building LoopBack
-applications is not yet fully supported. Therefore, some of the sections in this
-page are outdated and may not work out of the box. They will be revisited after
-our MVP release.
-" %}
-
 LoopBack 4 is more than just a framework: It’s an ecosystem that encourages
-developers to follow best practices through predefined standards. This section
-will walk through some important guidelines by building an example API for a
-catalog of products.
+developers to follow best practices through predefined standards.
+
+In this section, we provide guidelines on building LoopBack application and
+extensions the test-driven way, to make such projects robust and easy to
+maintain.
 
-Our best practice follows an "API first" and test-driven development approach:
+1.  [**Defining the API using code-first approach**](./Defining-the-API-using-code-first-approach.md):
```

Comment:
I think we should change the listing from numberings to bullets. The content here has no logical flow from one another and I think it'd be misleading to number these.